src/config/Config.zig

```diff
@@ -985,6 +985,14 @@ palette: Palette = .{},
 /// Available since: 1.1.0
 @"split-divider-color": ?Color = null,
 
+/// Control when Ghostty preserves the zoomed state of a split. This is a packed
```

Comment:
Seems like we are missing an `Available since:` annotation. Also, the doc string seems to leak Zig internals: "This is a packed struct...".

Reply:
Good point, let me follow that up real quick.